README.md

```diff
@@ -322,6 +323,25 @@ typedef NS_ENUM(NSInteger, NYTAdRequestState) {
 };
 ```
 
+## Boxing Enumerated Types
+
+Althought `enum`s are integral values, they cannot be directly used as [boxed literals](http://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-enums). So when creating a dictionary using `enum`s, they should be written as boxed literals instead of their corresponding `NSNumber` expression. 
+
+**For example:**
+```objc
+typedef NS_ENUM(NSInteger, NYTDaysOfTheWeek){
+     NYTMonday = 0, 
+     NYTTuesday
+}; 
+
+NSDictionary *daysOfTheWeek = @{@"monday" : @(NYTMonday), @"tuesday" : @(NYTTuesday)};
```

Comment:
I would typically add whitespace between `@{` and the contents, and before the closing `}`. Same on [line 342](https://github.com/NYTimes/objective-c-style-guide/pull/116/files#diff-04c6e90faac2675aa89e2176d2eec7d8R342).